auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -1126,6 +1126,11 @@ def train(self):
             tb_writer = SummaryWriter(log_dir=ckpt_path)
             print(f"Writing Tensorboard logs to {tb_writer.log_dir}")
 
+            if mlflow_is_imported:
+                mlflow.set_tracking_uri(os.path.join(ckpt_path, "mlruns"))
```

Comment:
Can we update this to accept `mlflow_tracking_uri` instead of local file storage, similar to what Dong has done for dints? Thanks

Reply:
Please see my comment in the PR. Thanks.